go/batch-submitter/drivers/sequencer/driver.go

```diff
@@ -187,6 +188,7 @@ func (d *Driver) CraftBatchTx(
 			// size also adheres to this constraint.
 			txLen := batchElement.Tx.Size()
 			if totalTxSize+uint64(TxLenSize+txLen) > d.cfg.MaxTxSize {
+				hasLargeFinalElement = totalTxSize < d.cfg.MinTxSize
```

Comment:
Perhaps we should call this `shouldIgnoreMinTxSize`? The current name is a bit confusing because `hasLargeFinalElement` is true only when the compressed batch is too small.

Reply:
yes band could be improved, it would be more accurate to say the _next_ element is large. can change this around